### DISCUSSION TITLE
Possibility to enable old text font again?

### DISCUSSION THREAD

**Discussion by ww123ist** [bcholmes/StarTrek2d20#205] [Ideas]
Hello! While I appreciate the new text font being more accessible to different languages, it is extremely hard for me or my fellow players to read (especially for the 2-page TNG portrait PDF which I find the most helpful.) The old font was nice and clear to read, while this new one requires extensive zoom-ins and everyone is unhappy and taking twice as long to read their own sheets, stalling play.

Could you perhaps add a checkbox at the final page to choose between fonts? Or maybe just have a selection of duplicate PDFs, one version with each font.

**Comment by bcholmes** [comment]
I've done a tweak of the 2-page portrait sheet (it was one of the ones that I hadn't revised when I switched over to the new font): does that help you out at all?

**Comment by ww123ist** [comment]
I think it's somehow even worse. I'm providing you with a visual so you can see the past copy and present, and you will see the difference. Even the coloring is now a washed-out gray from its previous black. Again this is for the TNG 2-pager portrait complete with the combat help sheet on the second page.
![CharSheetSTA](https://github.com/bcholmes/StarTrek2d20/assets/151906477/0eed818e-2e1f-4c21-9ef6-fa0a4d694d76)

**Reply by bcholmes** [reply]
Ah. Thank you for posting that. That's a different one than I thought we were talking about. You said the 2-page portrait, and that's a landscape sheet. Let me take a look.

**Reply by ww123ist** [reply]
Oh my mistake, sorry, slip of the tongue! I appreciate your assistance and all the hard work you've done!
